<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Termometro</class>
 <widget class="QMainWindow" name="Termometro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Termometro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/icono</normaloff>:/icono</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Fahrenheit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Fahrenheit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Centigrados">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-130</y>
      <width>161</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Centígrados &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>160</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="grpcenti">
     <item>
      <widget class="QLCDNumber" name="lcdCent"/>
     </item>
     <item>
      <widget class="QDial" name="dialCent">
       <property name="minimum">
        <number>-18</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>160</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="grpFah">
     <item>
      <widget class="QLCDNumber" name="lcdFahr"/>
     </item>
     <item>
      <widget class="QDial" name="dialFahr">
       <property name="maximum">
        <number>210</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>160</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="grpKelvin">
     <item>
      <widget class="QLCDNumber" name="lcdKelvin"/>
     </item>
     <item>
      <widget class="QDial" name="dialKelvin">
       <property name="minimum">
        <number>-18</number>
       </property>
       <property name="maximum">
        <number>370</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Kelvin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>158</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Kelvin &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Centi2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>158</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Centígrados&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>160</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="grpcent2">
     <item>
      <widget class="QLCDNumber" name="lcdcent2"/>
     </item>
     <item>
      <widget class="QDial" name="dialCent2">
       <property name="minimum">
        <number>-274</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Fahr2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Fahrenheit&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Kelvin2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>158</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Typewriter</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Kelvin &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="grpKelvin2">
     <item>
      <widget class="QLCDNumber" name="lcdKelvin2"/>
     </item>
     <item>
      <widget class="QDial" name="dialKel2">
       <property name="maximum">
        <number>310</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="grpFahr2">
     <item>
      <widget class="QLCDNumber" name="lcdFahr2"/>
     </item>
     <item>
      <widget class="QDial" name="dialFahr2"/>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>20</width>
      <height>631</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>275</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialCent2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdcent2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialKelvin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdKelvin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialKel2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdKelvin2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialFahr2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFahr2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
